EIPS/eip-2364.md

```diff
@@ -41,7 +42,7 @@ Whenever two peers connect using the `eth/64` protocol, the updated `Status` mes
 
 ## Rationale
 
-The specification is tiny since most parts are already specified in EIP-2124. `eth/63` is not specified as an EIP, but is maintained [here](https://github.com/ethereum/devp2p/blob/master/caps/eth.md).
+The specification is tiny since most parts are already specified in EIP-2124. `eth/63` is not specified as an EIP, but is maintained in the `ethereum/devp2p` Github repository.
```

Comment:
I know @MicahZoltu has a thing against links, but this "non-link link" to me feels beyond pointless.

Reply:
Since the repository doesn't have tags, and the argument is that these repositories may change their structure, can we at least agree on pointing to https://github.com/ethereum/devp2p (as a proper link), because:
1) Repositories rarely are deleted, but rather renamed
2) archive.org can be used to find a copy
3) If that repository gets deleted and no information is to be found about what happened to the specification, then it is reasonable to believe the EIPs repository and other specification are useless too.

Reply:
No strong opinion from me. Happy to change it to whatever is best.

Reply:
If it was only up to me, I'd pragmatically just allow links where there is no better alternative (which is the case here) 😅

Reply:
I don't personally see the value in that paragraph in the context of a technical specification.  Just like we don't have every core EIP link to the yellow-paper, I think it doesn't make sense to have every Networking EIP link to the devp2p repository.

The size of the changeset is inconsequential IMO and not something that we need to spend space on the EIP explaining.  Small EIPs are generally always better, and we should never need to feel the need to defend them.  In fact, I would weakly argue that you should have to defend *large* EIPs and explain why it can't be broken up into smaller EIPs, but not the other way around.

Reply:
Given these specs are already deprecated, can we optimize for merge speed and just remove the link :-) ?

Reply:
This is my preferred solution:
```suggestion
The specification is tiny since most parts are already specified in EIP-2124. `eth/63` is not specified as an EIP, but is maintained in the [ethereum/devp2p](https://github.com/ethereum/devp2p) Github repository.
```

Reply:
My preferred solution:
```suggestion
```

Reply:
As a reminder, despite appearances the EIP process is not actually a dictatorship.  If you want me to merge, remove the line.  If you want Alex to merge, change the line.  If you want something else, bribe @lightclient to allow some third option.  😆